src/HealthChecks.Uris/UriHealthCheckOptions.cs

```diff
@@ -134,6 +144,12 @@ public UriHealthCheckOptions ExpectHttpCodes(int minCodeToExpect, int maxCodeToE
             return this;
         }
 
+        public UriHealthCheckOptions ExpectContent(string expectedContent)

```

Comment:
> I think you missed the right place to configure. See UriHealthCheckOptions.AddUri and remove UriHealthCheckOptions.ExpectedContent from "root" options class.

Reply:
ping @roblohmann 

I would like this PR to be merged after some final fixes.

Reply:
@sungam3r appologies for the delay. Just changed this